fe/fe-core/src/main/java/org/apache/doris/nereids/rules/exploration/mv/MaterializedViewUtils.java

```diff
@@ -740,26 +952,30 @@ public static final class RelatedTableInfo {
         private final boolean pctPossible;
```

Comment:
All Related TableInfo should share one pctPossible,

Reply:
Have fixed it